syntax = "proto2";

message BrotherDecoderProto {}

enum BrotherFormat {
	BROTHER240 = 0;
	BROTHER120 = 1;
};

message BrotherEncoderProto {
	optional double clock_rate_us = 1 [default = 3.83];
	optional double post_index_gap_ms = 2 [default = 1.0];
	optional double sector_spacing_ms = 3 [default = 16.2];
	optional double post_header_spacing_ms = 4 [default = 0.69];

	optional BrotherFormat format = 6 [default = BROTHER240];
}

